steps/s1_build_segments.py

```diff
@@ -102,3 +102,15 @@ def _return_superpixels(im2arr, image_shape, resolutions):
             patches.append(patch)
 
         return superpixels, patches
+
+def segment_an_image(image, configuration):
```

Comment:
I knew there is a little bit of duplication here and other places. But I don't want to mess around with the train source code(Don't want to run train.py to make sure it still works 😢 ). Forgive me...